tensorflow/cc/framework/cc_op_gen.cc

```diff
@@ -57,6 +57,16 @@ string GetPath(const string& dot_h_fname) {
   return result;
 }
 
+// Converts: some/path/to/file.xx
+// to: file
+// (note that suffix is removed)
+string GetFilename(const string& path) {
```

Comment:
I thought we had no code changes among these?
This will be misleading, as these wont be in the binary, but it will be in the code of the release.